spec/controllers/polls_controller_spec.rb

```diff
@@ -3,7 +3,11 @@
 
 RSpec.describe PollsController, type: :controller do
   describe 'GET #index' do
-    let(:poll) { FactoryGirl.create(:valid_poll) }
+    before(:each) do
```

Comment:
Please, think about using [contexts](https://relishapp.com/rspec/rspec-core/v/3-4/docs/example-groups/basic-structure-describe-it#nested-example-groups-(using-`context`)) and [let syntax](https://relishapp.com/rspec/rspec-core/v/3-4/docs/helper-methods/let-and-let) here.
More generally, http://betterspecs.org/ guides are your friends.

It's mandatory — and I will LGTM the pullrequest without it. But it's a really powerful technique, definitely worth knowing about.